.github/workflows/ci.yml

```diff
@@ -8,64 +8,62 @@ jobs:
-
+  
+   # - name: Run the end to end test
+   #   run: docker run -e FLASK_APP=${{ secrets.FLASK_APP }} -e FLASK_ENV=${{ secrets.FLASK_ENV }} -e SECRET_KEY=${{ secrets.SECRET_KEY }} -e TRELLO_API_KEY=${{ secrets.TRELLO_API_KEY }} -e TRELLO_TOKEN=${{ secrets.TRELLO_TOKEN }} -e TRELLO_BOARD_ID=${{ secrets.TRELLO_BOARD_ID }} -e ID_LIST_TODO=${{ secrets.ID_LIST_TODO }} -e ID_LIST_DONE=${{ secrets.ID_LIST_DONE }} -e ID_LIST_DOING=${{ secrets.ID_LIST_DOING }} -e CARDS_TRELLODONE_CARD=${{ secrets.CARDS_TRELLODONE_CARD }} -p 5100:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app  todo-app:test /app/todo_app/SeleniumTest
 
     -  name: Build and push
        id: docker_build
-       uses: docker/build-push-action@v2
+       uses: docker/build-push-action@v2 
        with:
+          target: production
           push: true
-          tags: arpithadockerhub/app:latest
-         # tags: arpithadockerhub/dotnettemplate:latest
+          tags: arpithadockerhub/todo-app:latest 
+
+        
+    - uses: akhileshns/heroku-deploy@v3.12.12    #  This is the action
+      with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_app_name: "herukudevopsapp"
+          #Must be unique in Heroku
+          heroku_email: ${{secrets.HEROKU_EMAIL}} 
+          justlogin: true
+     
+    - name: "Login to heroku registry"
+      run: heroku container:login
+
+    - name: Docker tag
+      run: docker tag arpithadockerhub/todo-app:latest registry.heroku.com/herukudevopsapp/web
 
-# - uses: akhileshns/heroku-deploy@v3.12.12 # This is the action
-    #   with:
-    #       heroku_api_key: ${{secrets.HEROKU_API_KEY}}
-    #       heroku_app_name: "herukudevopsapp" #Must be unique in Heroku
-    #       heroku_email: ${{secrets.HEROKU_EMAIL}} 
-    #       justlogin: true
+    - name: Push docker image to Heroku 
+      run: docker push registry.heroku.com/herukudevopsapp/web
 
-    # - name: "Login to heroku registry"
-    #   run: heroku container:login
+    - name: "Releases the container"
+      run: heroku container:release web --app herukudevopsapp
 
-    # - name: "Build and push to heroku registry"
-    #   run: heroku container:push web --app herukudevopsapp
 
-    # - name: "Releases the container"
-    #   run: heroku container:release web --app herukudevopsapp
+ 
+
 
 
       # heroku container:push web --app herukudevopsapp
       # heroku container:release web --app herukudevopsapp
       # heroku logs --tail --app herukudevopsapp
       # docker tag arpithadockerhub/dotnettemplate  registry.heroku.com/herukudevopsapp/web
```

Comment:
There's quite a lot of commented out code in this file, we should clean it up and make it easier to read - either delete it or uncomment it

Reply:
Deleted comments

Reply:
deleted